starport/services/networkbuilder/networkbuilder.go

```diff
@@ -163,15 +163,30 @@ func (b *Builder) StartChain(ctx context.Context, chainID string, flags []string
 	if err != nil {
 		return err
 	}
-	launchInformation, err := b.spnclient.ChainGet(ctx, account.Name, chainID)
+
+
+	// reset the genesis in its initial form to ensure a deterministic generation
+	homedir, err := os.UserHomeDir()
 	if err != nil {
 		return err
 	}
 
-	homedir, err := os.UserHomeDir()
+	initialGenesisPath := filepath.Join(homedir, app.ND(), "config/initialGenesis.json")
```

Comment:
I would rather to use [os.Replace](https://golang.org/pkg/os/#Rename) here instead of reading/writing current and initial genesis files.

Reply:
There is no `os.Replace`